### PR TITLE
nlopt: add details

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: nlopt/build.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
-./configure --prefix=$PREFIX --enable-shared
+./configure --prefix=$PREFIX --enable-shared --without-octave --without-matlab --without-guile --without-python
```
Comment (ccordoba12) [inline nlopt/build.sh]: why `--without-python`? I think we should leave at least that one :-)

**Comment by jschueller** [comment]
@ccordoba12, it seems not consistent with the windows side, moreover it should depend on python runtime to distinguish the python version. As-is it should not work as we should specify the python location to avoid using host configuration.

**Comment by jschueller** [comment]
@ccordoba12, here you go, proper python bindings

**Comment by ccordoba12** [comment]
Thanks a lot!

**Pull request merged**